### PR TITLE
Add proposal for thin-plate splines Outreachy Project

### PR DESCRIPTION
**Pull request by lagru**
Content copied from https://www.outreachy.org/outreachy-may-2023-internship-cohort/communities/scikit-image/enhance-image-warping-in-scikit-image-with-thin-pl/cfp/ (might only be visible with login).

@mkcor, feel welcome to merge once you get around to it. :)

### CODE REVIEW

File: internships/Outreachy_2023-05/Project_TLP_proposal.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Outreachy proposal on `transform` module and thin-plate spines
```
Comment (mkcor) [inline internships/Outreachy_2023-05/Project_TLP_proposal.md]:
```suggestion
# Outreachy proposal on `transform` module and thin-plate splines

```

File: internships/Outreachy_2023-05/Project_TLP_proposal.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+### System Requirements
+The intern will need one of the following platforms:
+- Windows 64-bit on x86 processors
+- Mac OS X on x86 / Apple Silicon / M1 / M2 processors
+- Linux 64-bit on x86 processors
+
+### Project Description
+
+**Title:** Enhance image warping in scikit-image with thin-plate spines
```
Comment (mkcor) [inline internships/Outreachy_2023-05/Project_TLP_proposal.md]:
```suggestion
**Title:** Enhance image warping in scikit-image with thin-plate splines
```

File: internships/Outreachy_2023-05/Project_TLP_proposal.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+Depending on the progress during the internship, the intern may also optimize the performance of the algorithm and create a new tutorial or gallery example for the new function.
+
+### Intern Benefits
+The intern will experience the end-to-end process from conception of a new idea to its integration into an existing API. With the help of mentors and the community, the intern may expand their skill set in a wide range of concepts and technologies. This will involve API design, code maintainability, performence optimization, and effective collaboration on these things. 
+
+Apart from personal improvement, the intern will also engage with and become part of a welcoming, skilled, international and science focused community that values openness, collaboration and diversity. The mentors and the scikit-image team would be very happy to see new opportunities develop following the internship.
+
+### Community benefits
```
Comment (mkcor) [inline internships/Outreachy_2023-05/Project_TLP_proposal.md]:
```suggestion
### Community Benefits
```
(just to match the above)

File: internships/Outreachy_2023-05/Project_TLP_proposal.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  [Follow this link](https://skimage.zulipchat.com/) to join this project's public chat.
+Log in with GitHub or create a new account. Join topic ["Outreachy" under stream "sprint"](https://skimage.zulipchat.com/#narrow/stream/176555-sprint/topic/Outreachy). Please introduce yourself and post there! You are welcome to tag your mentor (Lars Grüter @lagru) or send him a private message if you have questions.
+- **Developer forum** - You can find helpful documentation about this communication tool [here](https://meta.discourse.org/t/discourse-new-user-guide/96331). Once you join the project's communication channel, the mentors have some additional instructions for you to follow:
+  This is a good place for longer-form discussions, project-wide topics, decision making and more general discussion of contributions. Before creating a topic here, please search for similar or related ones first and create it in "Contributor & Developer discussion" » "scikit-image" or another appropriate category.
+  [Follow this link](https://discuss.scientific-python.org/c/contributor/skimage/22) to join this project's public chat.
```
Comment (mkcor) [inline internships/Outreachy_2023-05/Project_TLP_proposal.md]:
```suggestion
  [Follow this link](https://discuss.scientific-python.org/c/contributor/skimage/22) to join this project's public forum.
```

File: internships/Outreachy_2023-05/Project_TLP_proposal.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+Log in with GitHub or create a new account. Feel welcome to tag your mentor (Lars Grüter @lagru) or send him a private message to introduce yourself and ask questions.
+
+- **GitHub** - You can find helpful documentation about this communication tool [here](https://docs.github.com/en). Once you join the project's communication channel, the mentors have some additional instructions for you to follow:
+  This is scikit-image's most formal communication space. We aim to keep issues, pull requests and comments focused on the specific development task at hand and on topic. Before creating a new issue or pull request here, please search for similar or related ones first. If in doubt whether post a topic here, you are very welcome to ask on our other communication channels first (Developer forum, Zulip). Also, feel free to tag your mentor (Lars Grüter @lagru) at anytime.
+  [Follow this link](https://github.com/scikit-image/scikit-image) to join this project's public chat.
```
Comment (mkcor) [inline internships/Outreachy_2023-05/Project_TLP_proposal.md]:
```suggestion
  [Follow this link](https://github.com/scikit-image/scikit-image) to join this project's public chat.
```
```suggestion
  [Follow this link](https://github.com/scikit-image/scikit-image) to join this project's public repo.
```

**Review by mkcor — COMMENTED** [review]
Also, I would expect the filename to be `Project_TPS_proposal.md` rather than `Project_TLP_proposal.md` (right?).

**Comment by mkcor** [comment]
Good idea, @lagru! Thanks for creating this directory.

**Comment by lagru** [comment]
Thanks for the review! No idea how my brain arrived at "TLP" instead of TPS. :sweat_smile:

**Review by mkcor — APPROVED** [review]

**Pull request merged**